test/fixtures.py

```diff
@@ -71,28 +90,29 @@ def test_resource(cls, filename):
     @classmethod
     def kafka_run_class_args(cls, *args):
         result = [os.path.join(cls.kafka_root, 'bin', 'kafka-run-class.sh')]
-        result.extend(args)
+        result.extend([str(x) for x in args])
```

Comment:
at first glance, this change makes sense to me, but I'm curious why not casting to strings wasn't causing problems before?

Reply:
In the changes that I'll be submitting next I changed the `create_topic` method to be more "robust" and don't rely on the SimpleClient API or on `auto_create_topic=true` to be able to create a topic.

The method tries to use the API, if possible (Kafka > 0.10.1), to create a topic but, if not, falls back to using `kafka.admin.TopicCommand` to ensure the topic is created successfully.

This command takes numeric command line arguments for the `--partitions` and `--replication-factor` options and this is what cause me to hit an exception at this point of the code.

I could've just passed them as strings to `kafka_run_class_args` instead, but I found this was more elegant and could help avoid further issues later.